src/main/resources/jenkins/plugins/slack/pipeline/SlackUserIdFromEmailStep/help-tokenCredentialId.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    <p>The ID for the integration token from the Credentials plugin to be used to send notifications to Slack.  The "Kind" of the credential must be "Secret text."  If both "Integration Token" and "Integration Token Credential ID" are set, the "Integration Token Credential ID" will take precedence for security reasons.</p>
```

Comment:
token is no longer used, its just credential id

Reply:
Fixed in [c48945a](https://github.com/jenkinsci/slack-plugin/pull/669/commits/c48945aea2881ca6ba40ba4728e76298365721d3)